source/_posts/2023-06-06-php-core-roundup-13.md

```diff
@@ -0,0 +1,279 @@
+---
+title: 'PHP Core Roundup #13'
+layout: post
+tags:
+- roundup
+author:
+  name: ChatGPT
+  url: https://chat.openai.com/  
+published_at: 06 June 2023
+
+---
+
+The PHP Core team has been as productive as ever this past month, bringing forth a robust collection of updates that promise to shape the future of PHP. From RFCs that are sure to stir up lively debate, to ones that bring small, yet impactful changes, it has been a month filled with interesting developments. Here's what you need to know.
+
+> The PHP Foundation is a collective of PHP contributors, veterans, and companies that collaborate to ensure the long-term sustainability of the PHP programming language. The foundation currently supports six contributors focused on PHP's maintenance, debt reduction, and feature development. These contributors work closely with others on coding, documentation, and discussions.
+
+<br>
+
+> Started over a year ago, the PHP Core Roundup series offers a summary of the latest developments, discussions, and news about PHP Core, contributed by both PHP Foundation members and other participants. This post is the thirteenth in the PHP Core Roundup series.
+
+{% include "newsletter.html" %}
+
+## Releases
+
+The PHP development team released two new versions in May 2023: .
+
+**[PHP 8.2.6](https://www.php.net/archive/2023.php#2023-05-11-1)**
+- Security fixes: No specific CVEs were mentioned in the changelog.
+- Other changes: This release includes several bug fixes and improvements, notably in areas such as Core, Date, DOM, Exif, Intl, PCRE, Reflection, SPL, Standard, and Streams.
+
+**[PHP 8.1.19](https://www.php.net/archive/2023.php#2023-05-11-2)** 
+- Security fixes: No specific CVEs were mentioned in the changelog.
+- Other changes: This release includes bug fixes across various components such as Core, DOM, Exif, Intl, PCRE, and Standard.
```

Comment:
```suggestion
**[PHP 8.2.6](https://www.php.net/archive/2023.php#2023-05-11-1)**

This release includes several bug fixes and improvements, notably in areas such as Core, Date, DOM, Exif, Intl, PCRE, Reflection, SPL, Standard, and Streams.

**[PHP 8.1.19](https://www.php.net/archive/2023.php#2023-05-11-2)** 

This release includes bug fixes across various components such as Core, DOM, Exif, Intl, PCRE, and Standard.
```

8.2.6 and 8.1.19 – bug fix
security releases (8.0.29, 8.1.20, 8.2.7) will be announced today